src/sourcemaps/utils/sdk-version.ts

```diff
@@ -0,0 +1,213 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import { minVersion, satisfies } from 'semver';
+import {
+  abortIfCancelled,
+  getPackageDotJson,
+  getPackageVersion,
+  hasPackageInstalled,
+  installPackage,
+} from '../../utils/clack-utils';
+
+import * as Sentry from '@sentry/node';
+
+const MINIMUM_DEBUG_ID_SDK_VERSION = '7.47.0';
+
+const SENTRY_SDK_PACKAGES = [
```

Comment:
Do we want to add a short comment here saying that this array is sorted by order of detection?